compact-migrations

```diff
@@ -48,6 +48,19 @@ do
 	echo >> "${NEXT}"
 	git rm "${FILE}"
 done
+
+ls | grep "^V8.*" > dangling_migrations
+DANGLING_MIGRATION_COUNT=$(cat dangling_migrations | wc -l)
```

Comment:
Could just do

```
DANGLING_MIGRATION_COUNT=`ls | grep "^V8.*" | wc -l`
```

and then repeat the `ls | grep "^V8.*"` part below to avoid writing a file.

Or delete the file after

Reply:
Updated